index.html

```diff
@@ -538,6 +538,22 @@ <h2 id="changing_bundles_without_restart">Changing bundles in the OSGi container
 </div>
 
       </div>
+<div>      
+  <h2 id="google_analytics_tracking">Google Analytics tracking</h2>
+  <p>
+    The eosgi-maven-plugin collect anonym usage statistics. The collected statistics use to obtain budget to further development.
+    Collected data are:
+  </p>
+  <ul>
+    <li>analytics referer: use the eosgi-maven-plugin or other plugin. Example eclipse-e4-plugin, which is under development now.</li>
```

Comment:
analytics referer: the eosgi-maven-plugin or other plugin. Example eclipse-e4-plugin, which is under development now.